app/utils/latest-action.js

```diff
@@ -1,13 +1,13 @@
 import moment from 'moment';
 
 
-export default function latestAction(actions) {
-    if (!actions.length) {
+export default function latestAction(submissionActions) {
```

Comment:
Out of curiosity, why the rename?

Reply:
it was part of resolving conflicts with Lauren's changes. But it is weird to show as change by me.